src/pages/Home/map/LeaguesMap.tsx

```diff
@@ -0,0 +1,352 @@
+import { useState } from "react";
+import GoogleMapReact from "google-map-react";
+import MapDrawer from "./MapDrawer";
+import { FlexDiv } from "../../../react-design-system/FlexDiv";
+import { useSelector } from "react-redux";
+import { RootState } from "../../../interfaces/redux/store";
+import img from "../../../../src/assets/location.png";
+import { useNavigator } from "../../../costumHooks/currentLocation";
```

Comment:
I think we can move all hooks to utils folder what do you think

Reply:
basically, we can  make more custom hooks for fetching data, pagination, and much more as our requirement so we left it as separate entity then it may be okay.